libs/dyn/mapping_test.go

```diff
@@ -15,7 +15,7 @@ func TestNewMapping(t *testing.T) {
 }
 
 func TestMappingZeroValue(t *testing.T) {
-	var m dyn.Mapping
+	m := dyn.NewMapping()
```

Comment:
Is the zero-initialized value no longer usable?

Reply:
It was not. Restored it so it's usable again now. c7478c809016b38f2f1acb35a2f4412311ff35b1